packages/grafana-ui/src/components/Table/LogViewCell.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { useStyles2 } from '../../themes';
+import { TableCellProps } from './types';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export function LogViewCell(props: TableCellProps): JSX.Element {
+  const { cell, tableStyles, cellProps } = props;
+  return (
+    <Tooltip placement="auto-start" content={<LogTooltip value={cell.value} />} theme="info-alt">
```

Comment:
theme should not be added here, (info-alt should not be used, deprecated, not showing correct tooltip color)